vec120/src/main/java/com/foursoft/vecmodel/vec120/navigations/SegmentNavs.java

```diff
@@ -50,7 +50,6 @@ private SegmentNavs() {
         return segment -> segment.getParentTopologySpecification().getParentDocumentVersion().getDocumentNumber();
     }
 
-    // TODO Make the segmentLengthClassification an Open Enumeration
```

Comment:
Why was that ToDo removed?

Reply:
not sure if we're doing that in this project. I think also in such cases issues are better than todos

Reply:
Agree on that, I would open an issue then.